### ISSUE TITLE
Breadcrumb - refactoring and a11y

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#3821]
The Breadcrumb component should be refactored so it has a proper markup and a11y features. 
- use Ordered List
- aria-label, aria-current
- keyboard navigation - tabs, arrows

a11y guidelines: https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html

**Comment by InnaAtanasova** [comment]
part of https://github.com/SAP/fundamental-styles/issues/3962
